src/Components/CreateTask.js

```diff
@@ -0,0 +1,149 @@
+    newTask.set("status", "To do")
+    newTask.set("section", "Culture")
+    newTask.set("description", description);
+
+    try {
+      await newTask.save();
+      alert("succes");
+    } catch (error) {
+      alert(error);
+    }
+  }
+
+  return (
+    <>
+      <Container className="container">
+        <Form>
+          <Row className="upperrow">
+          <Col>
+            <TitleForm text="Task Title" setTitle= {setTitle} />
+            </Col>
+
+            <Col>
+            <TitleForm text="Task Description" setTitle= {setDescription} />
+
+            </Col>
+
+            <Col lg="3">
+              <Form.Group className="formpart" controlId="formStatusSelection">
+                <Form.Label>Status</Form.Label>
+                <Form.Group>
+                  {" "}
+                  {/** right approach? */}
+                  <Form.Select defaultValue="Select">
+                    <option>Select</option>
+                    <option>to Do</option>
+                    <option>Doing</option>
+                    <option>Done</option>
+                  </Form.Select>
+                </Form.Group>
+              </Form.Group>
+            </Col>
+          </Row>
+
+          <Row className="lowerrow">
+            <Col lg="3">
+              <Form.Group className="formpart" controlId="formSectionSelection">
+                <Form.Label>Section</Form.Label>
+                <Form.Group>
+                  <Form.Select className="mb-4" defaultValue="Select">
+                    <option>Select</option>
+                    <option>Culture</option>
+                    <option>Finance</option>
+                    <option>Politics</option>
+                    <option>Travel</option>
+                  </Form.Select>
+                </Form.Group>
+              </Form.Group>
+            </Col>
+
+            <Col>
+              <Form.Group
+                className="formpart"
+                controlId="formPlainTextResponsible"
+              >
+                <Form.Label>Responsible</Form.Label>
+                <Container className="row">
+                  <Form.Control
+                    style={{ width: "35px" }}
+                    plaintext
+                    readOnly
+                    defaultValue="You"
+                    onChange={(e) => setResponsible(e.target.value)}
+                  />
+                  <Button
+                    className="resbutton"
+                    style={{
+                      display: "flex",
+                      justifyContent: "center",
+                      alignItems: "center",
+                      paddingInline: "0px",
+                      background: "#FFF",
+                      color: "#1C1C1C",
+                      borderColor: "#1C1C1C",
+                      borderRadius: "20px",
+                      border: "solid 2px",
+                      fontSize: "1.5rem",
+                      height: "1.5rem",
+                      width: "1.5rem",
```

Comment:
Inline styling is always a bad idea, especially when you combine with stylesheets elsewhere. You gave it a class name. Use that :D

Reply:
(I know why it isn't that easy. The fact that it isn't that easy should tell you why this is a less than optimal way of doing it. Priorities, of course, I know.)